pegasus/test/test_i18n_hoc_routes.rb

```diff
@@ -0,0 +1,37 @@
+require 'rack/test'
+require 'minitest/autorun'
+
+class I18nHocRoutesTest < Minitest::Test
+  include Rack::Test::Methods
+
+  def app
+    Rack::Builder.parse_file(File.absolute_path('../config.ru', __dir__)).first
+  end
+
+  def test_hoc_pages_for_all_locales
+    header 'Host', 'hourofcode.com'
+    languages = DB[:cdo_languages].select(:unique_language_s).where(supported_hoc_b: 1)
+    subpages = load_hoc_subpages
+
+    languages.each do |prop|
```

Comment:
Oh I think I can simplify this since I'm just referencing one property like you mentioned before